src/test/storage/adaptive_radix_tree_index_test.cpp

```diff
@@ -140,7 +176,62 @@ TEST_F(AdaptiveRadixTreeIndexTest, VectorOfRandomInts) {
       expected_lower += 2;
     }
   }
-  EXPECT_EQ(index->upper_bound({99999}), index->cend());
+
+  for (int search_item = 0; search_item < static_cast<int>(3 * test_size); ++search_item) {
```

Comment:
I think a comment for the loop condition could be useful.

Reply:
Added a comment.